app/src/main/java/lt/vilnius/tvarkau/FullscreenImageActivity.kt

```diff
@@ -0,0 +1,68 @@
+package lt.vilnius.tvarkau
+
+import android.os.Bundle
+import android.view.MenuItem
+import android.view.View
+import kotlinx.android.synthetic.main.activity_fullscreen.*
+import lt.vilnius.tvarkau.views.adapters.FullscreenImagesPagerAdapter
+
+class FullscreenImageActivity : BaseActivity() {
+
+    private var initialImagePosition: Int = 0
```

Comment:
This approach works better - you can see where value is retrieved from:

```
private val initialImagePosition: Int
  get() = intent.getIntExtra(....)
```

Same with `photos`